components/pushbullet/pushbullet.app.mjs

```diff
@@ -1,11 +1,125 @@
+import { axios } from "@pipedream/platform";
+import FormData from "form-data";
+
 export default {
   type: "app",
   app: "pushbullet",
-  propDefinitions: {},
+  propDefinitions: {
+    device: {
+      label: "Device",
+      description: "Select a device",
+      type: "string",
+      async options() {
+        const devices = await this.getDevices();
+
+        return devices.map((device) => ({
+          label: device.nickname,
+          value: device.iden,
+        }));
+      },
+    },
+    push: {
+      label: "Sended Push",
+      description: "Select a push",
+      type: "string",
+      async options() {
+        const pushes = await this.getPushes();
+
+        return pushes.map((push) => ({
+          label: push.title ?? push.iden,
+          value: push.iden,
+        }));
+      },
+    },
+  },
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    _accessToken() {
+      return this.$auth.oauth_access_token;
+    },
+    _apiUrl() {
+      return "https://api.pushbullet.com/v2";
+    },
+    async _makeRequest(path, options = {}, $ = undefined) {
```

Comment:
You can use `$=this`, so you don't need to use `$ ?? this`. In this case `$ = undefined` is redundant